io.github.SkiffDesktopAdw.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id" : "io.github.SkiffDesktopAdw",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+
+    ],
+    "command" : "io.github.SkiffDesktopAdw",
+    "finish-args" : [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--device=dri",
+        "--share=network"
```

Comment:
does this really need Network access?

Reply:
Yes. The wrapper needs internet access to login to Skiff.com and access email and such. Thank you.